graphbolt/src/csc_sampling_graph.cc

```diff
@@ -97,6 +112,11 @@ void CSCSamplingGraph::Save(torch::serialize::OutputArchive& archive) const {
   if (type_per_edge_) {
     archive.write("CSCSamplingGraph/type_per_edge", type_per_edge_.value());
   }
+  archive.write(
```

Comment:
Why we need this Bool? just save it if present otherwise do nothing.

Reply:
Firstly, this ensures consistency with the existing code. Secondly, I believe it simplifies the logic for `Load`. Using this to indicate the presence of `edge_attributes_` aligns with the concept of flags in TCP headers, streamlining the code.